tests/e2e/test_smoke.py

```
@@ -250,12 +250,292 @@ def accept_delete(dialog) -> None:
+        )
+        assert sample["gridContainer"] == ref["gridContainer"], (
+            f"grid-container geometry differs at {count}: "
+            f"got {sample['gridContainer']}, expected {ref['gridContainer']}"
+        )
+        assert sample["svg"] == ref["svg"], (
+            f"grid-svg geometry differs at {count}: "
+            f"got {sample['svg']}, expected {ref['svg']}"
+        )
+        for ref_tile, sample_tile in zip(
+            ref["tilePlatformPositions"], sample["tilePlatformPositions"]
+        ):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)